Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -109,7 +109,9 @@ def set_data(self, data):
             self.varlist[:] = [var for var in data.domain.variables
                                if var.is_discrete]
             self.selected_var_indices = [0, 1][:len(self.varlist)]
-            self.component_x, self.component_y = 0, 1
+            self.component_x = 0
+            self.component_y = 1 if \
+                len(self.varlist[self.selected_var_indices[-1]].values) > 1 else 0
```

Comment:
He does need len, because he wants to check if it is >1 and not >0.
It is also harder to annoy @astaric, but the `if ... else` is unnecessary, so:
`self.component_y = len(self.varlist[self.selected_var_indices[-1]].values) > 1`
or if someone is the strict type:
`int(len(self.varlist[self.selected_var_indices[-1]].values) > 1)`